### ISSUE TITLE
<speaker> as content of <lem>

### ISSUE THREAD

**Issue by gabrielegan** [TEIC/TEI#2316]
A gap in a manuscript -- that is, physical damage causing some paper to be missing -- may create a situation that an editor wants to discuss in a textual note auto-created from an <app> element. The lost words in the manuscript may or may not have included a speech prefix, so it should be allowed that the element <speaker> for a speech prefix that the editor thinks was there (but is now lost) may appear within a <lem> or a <rdg> element, like this:

    <sp who="#SURESBY">
      <app type="textual">
        <lem wit="#SHIRLEY">
          <speaker>[Sure.]</speaker>
        </lem>
        <rdg wit="#MS"><hi rend="italic">writing
              presumed lost to damage</hi></rdg>
      </app>
      <l>as familiare as thou wilt my knaue</l>
      <l>tis this I long to knowe.</l>
    </sp>

But the above is not valid by the current TEI rules. I propose that the above kind of construction should be valid, since a lemma or a reading may indeed contain a speech prefix.

**Comment by raffazizzi** [comment]
The problem is that it's not really possible with our current tools to limit the content model of `<lem>` and `<rdg>` when their ancestor is `<sp>` so if we allowed `<speaker>` in `<lem>` and `<rdg>`, it would make it available in other contexts where it doesn't belong.

Would the approach below work for you? It encodes there's a speaker, actual or presumed, and encodes the content as separate readings:

```xml
<sp who="#SURESBY">
   <speaker>
      <app type="textual">
         <lem wit="#SHIRLEY">[Sure.]</lem>
         <rdg wit="#MS"><gap reason="lost"/></rdg>
      </app>
   </speaker>
   
   <l>as familiare as thou wilt my knaue</l>
   <l>tis this I long to knowe.</l>
</sp>
```

Also note that I replaced your editorial text in `<hi>` with a more semantic `<gap>`, but that's just a suggestion.

**Comment by hcayless** [comment]
I'm not persuaded by the argument that `<app>` is a backdoor to doing inappropriate things. But I think Raff has a point in this case. For completeness, I'd also replace the brackets with markup:
```xml
<sp who="#SURESBY">
   <speaker>
      <app type="textual">
         <lem wit="#SHIRLEY"><supplied reason="lost">Sure.</supplied></lem>
         <rdg wit="#MS"><gap reason="illegible"/></rdg>
      </app>
   </speaker>
   
   <l>as familiare as thou wilt my knaue</l>
   <l>tis this I long to knowe.</l>
</sp>
```
If you really want the note, it can be supplied in a `<witDetail>` or `<note>`.

**Comment by gabrielegan** [comment]
@raffazizzi Yep, what you suggest is what we settled on doing, but it's an encoding that asserts more than we want to assert. That is, we don't know that there was a speech prefix at this point in the manuscript, since a bit of paper (which may or may not have contained a speech prefix) has broken off from the manuscipt and been lost. We'd prefer the `<speaker>` element to be inside the `<lem>` element so that the very claim that there was a speech prefix here is part of the editorial hypothesis of the editorial authority we cite as SHIRLEY.

Also, surely the `<speaker>` element does not assert that there is a speaker (the `<sp>` element and its @who attribute does that), as you say, but rather asserts that there is a speech prefix for that speaker. There's definitely a speaker but we don't know for sure who the speaker is. 

Or have I misunderstood what `<sp>` and `<speaker>` assert?

**Comment by martindholmes** [comment]
Council F2F agreed that this should be done.

**Comment by hennyu** [comment]
This is done and closed @gabrielegan .

**Comment by ebeshero** [comment]
Just adding reference to the commit that closes this issue:
https://github.com/TEIC/TEI/pull/2807
